plugins/main/public/components/common/data-source/hooks/use-data-source.ts

```diff
@@ -11,8 +11,10 @@ import {
   PatternDataSourceFilterManager,
   tFilterManager,
 } from '../index';
+import { TimeRange } from '../../../../../../../src/plugins/data/public';
 import { PinnedAgentManager } from '../../../wz-agent-selector/wz-agent-selector-service';
 import { useIsMounted } from '../../hooks/use-is-mounted';
+import { transformDateRange } from '../../search-bar/search-bar-service';
```

Comment:
`transformDateRange` should not be imported as it is not used. Right?

Reply:
Solved in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6923/commits/bb1090875952758308c502a73bf2b80e2ad6f1b0